packages/server-web/src/client/component/TodoItem.vue

```diff
@@ -195,4 +197,10 @@ export default {
 .dropzone-sub.active-bottom {
   border-bottom: 5px solid lightgreen;
 }
+.dropzone-active {
+  z-index: 200;
```

Comment:
I'll open an issue for making our CSS code use variables. I think it would be nice to have a variable for this `z-index` here, especially when we might add modals due to some event while we are dragging (guess that should not happen, but who knows). I wouldn't like to have the dropzones over everything else then.